projects/1.18/assets/relics-mod/relics/lang/zh_cn.json

```
@@ -1 +1,287 @@
+  "tooltip.relics.enders_hand.ability.neutrality.description": "末影人对佩戴者中立",
+  "tooltip.relics.enders_hand.ability.swap": "末影交换",
+  "tooltip.relics.enders_hand.ability.swap.description": "交换佩戴者与看向的实体的位置。",
+  "tooltip.relics.enders_hand.ability.swap.stat.distance": "距离：%1$s格",
+
+  "item.relics.holy_locket": "神圣吊坠",
+  "tooltip.relics.holy_locket.leveling": "通过此遗物的能力每收取1点生命获得1点经验(收取的生命值计算时四舍五入)。",
+  "tooltip.relics.holy_locket.lore": "由各各他金属制作的闪光十字架。赐予正义的力量，制裁邪恶",
+  "tooltip.relics.holy_locket.ability.steal": "神圣祝福",
+  "tooltip.relics.holy_locket.ability.steal.description": "按一定百分比收取附近生物再生的生命值。",
+  "tooltip.relics.holy_locket.ability.steal.stat.amount": "数值：再生生命值的%1$s %%",
+  "tooltip.relics.holy_locket.ability.steal.stat.radius": "半径：%1$s格。",
+
+  "item.relics.magic_mirror": "魔镜",
+  "tooltip.relics.magic_mirror.leveling": "每次使用能力时获得1点经验，每传送50格额外获得1点经验。",
+  "tooltip.relics.magic_mirror.lore": "曾被强大先知附身的镜子，获得了跨维度旅行的能力。",
+  "tooltip.relics.magic_mirror.ability.teleport": "虫洞",
+  "tooltip.relics.magic_mirror.ability.teleport.description": "传送到佩戴者的重生点",
+  "tooltip.relics.magic_mirror.ability.teleport.stat.distance": "距离：%1$s格。",
+  "tooltip.relics.magic_mirror.ability.teleport.stat.cooldown": "冷却：%1$s秒。",
+
+  "item.relics.shadow_glaive": "暗影之刃",
+  "tooltip.relics.shadow_glaive.leveling": "环刃每进行2次弹跳获得1点经验",
+  "tooltip.relics.shadow_glaive.lore": "失落的末影文明为镇压叛乱而制造的武器。即使是创作者也未曾有机会对其进行测试。",
+  "tooltip.relics.shadow_glaive.ability.glaive": "盲目正义",
+  "tooltip.relics.shadow_glaive.ability.glaive.description": "使用时发射1枚在附近实体之间弹跳的环刃。",
+  "tooltip.relics.shadow_glaive.ability.glaive.stat.damage": "伤害：%1$s点",
+  "tooltip.relics.shadow_glaive.ability.glaive.stat.recharge": "充能时间：%1$s秒",
+  "tooltip.relics.shadow_glaive.ability.glaive.stat.bounces": "弹跳次数：%1$s",
+  "tooltip.relics.shadow_glaive.ability.glaive.stat.radius": "弹跳范围：%1$s格",
+  "tooltip.relics.shadow_glaive.ability.saw": "残酷惩戒",
+  "tooltip.relics.shadow_glaive.ability.saw.description": "消耗所有充能并暂时禁用恢复，召唤对附近实体造成伤害的巨型环刃。",
+  "tooltip.relics.shadow_glaive.ability.saw.stat.speed": "攻击速度：%1$s次/秒",
+  "tooltip.relics.shadow_glaive.ability.saw.stat.damage": "伤害：%1$s点",
+
+  "item.relics.arrow_quiver": "箭袋",
+  "tooltip.relics.arrow_quiver.leveling": "目标与佩戴者之间的距离每有10格获得1点经验",
+  "tooltip.relics.arrow_quiver.lore": "弓箭手必备。使用绝迹神秘野兽的毛皮，以精灵技术加工制作。",
+  "tooltip.relics.arrow_quiver.ability.receptacle": "箭矢存储",
+  "tooltip.relics.arrow_quiver.ability.receptacle.description": "射箭时优先使用箭袋中的箭。鼠标中有箭时左键点击箭袋可以将箭存入箭袋，而右键点击则将取出最近放入的箭。",
+  "tooltip.relics.arrow_quiver.ability.receptacle.stat.slots": "栏位：%1$s",
+  "tooltip.relics.arrow_quiver.ability.agility": "迅猛",
+  "tooltip.relics.arrow_quiver.ability.agility.description": "加快拉弓速度",
+  "tooltip.relics.arrow_quiver.ability.agility.stat.modifier": "加速倍率：%1$s %%",
+
+  "item.relics.blazing_flask": "炽热烧瓶",
+  "tooltip.relics.blazing_flask.leveling": "使用此遗物的能力每飞行5秒获得1点经验。",
+  "tooltip.relics.blazing_flask.lore": "猪灵文明的奇迹造物，在测试时将绯红森林变成了废土。",
```

```suggestion
  "tooltip.relics.blazing_flask.lore": "猪灵文明的奇迹造物，在测试时将绯红森林变成了荒地。",
```
W大写可能是指下界荒地（Nether Wastes）这个生物群系？
不确定（）